tests/trainer/test_lr_finder.py

```diff
@@ -271,3 +271,27 @@ def test_lr_finder_fails_fast_on_bad_config(tmpdir):
     trainer = Trainer(default_root_dir=tmpdir, max_steps=2, auto_lr_find=True)
     with pytest.raises(MisconfigurationException, match='should have one of these fields'):
         trainer.tune(BoringModel())
+
+
+def test_lr_find_with_bs_scale(tmpdir):
+    """ Test that lr_find runs with batch_size_scaling """
+
+    class BoringModelTune(BoringModel):
+        def __init__(self, hparams):
+            super().__init__()
+            self.save_hyperparameters(hparams)
+
+    hparams = EvalModelTemplate.get_default_hparams()
+    before_lr = hparams.get('learning_rate')
```

Comment:
Why don't you hardcode a `lr` in `BoringModelTune` instead of getting it from `EvalModelTemplate`?

Reply:
the hparams contains bs for bs scaling as well

Reply:
I mean something like


```python
    class BoringModelTune(BoringModel):
        def __init__(self, lr=0.01):
            super().__init__()
            self.lr = lr
            self.save_hyperparameters(hparams)

    before_lr = model.lr
```

Just to not rely on `EvalModelTemplate` at all

Reply:
Yeah can do that as well, just thought I'd reuse existing code since the rest of that script uses EvalModelTemplate. I'll update the PR as soon as I get a chance.